cmd/orb.go

```diff
@@ -1746,3 +1792,29 @@ func orbInformThatOrbCannotBeDeletedMessage() {
 	fmt.Println("Once an orb is created it cannot be deleted. Orbs are semver compliant, and each published version is immutable. Publicly released orbs are potential dependencies for other projects.")
 	fmt.Println("Therefore, allowing orb deletion would make users susceptible to unexpected loss of functionality.")
 }
+
+func (o *orbOptions) getOrgId(orgInfo orbOrgOptions) (string, error) {
+	if orgInfo.OrgID == "" && orgInfo.OrgSlug == "" {
+		return "", nil
+	}
+
+	var orgID string
+	if strings.TrimSpace(orgInfo.OrgID) != "" {
+		orgID = orgInfo.OrgID
```

Comment:
Nitpicking: you could return directly here, avoiding too much nesting with the following `else if`

Reply:
Addressed 👍